site/content/3.12/graphs/graph-visualizer.md

```diff
@@ -51,16 +52,16 @@ supported by the Graph Visualizer.
 
 ### Select and load a graph
 
-1. In the ArangoDB Platform web interface, select the database your named graph
+1. In the Data Platform web interface, select the database your named graph
    is stored in.
 2. Click **Graphs** in the main navigation.
 3. Select a graph from the list.
 4. The viewport of the Graph Visualizer opens for exploring the graph.
 
 The main area of the viewport may initially be empty in the following cases:
```

Comment:
This will change, the Visualizer will automatically populate the canvas with an initial neighbourhood, see https://github.com/arangodb/platform-ui/pull/217

Reply:
Not available in dev yet unfortunately - I would like to try this quickly before writing about it.

Reply:
This has just landed in dev